src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1854,6 +1854,42 @@ limitations under the License.
             <include cluster="Relative Humidity Measurement" client="false" server="false" clientLocked="false" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-fan</name>
+        <domain>HA</domain>
+        <typeName>Matter Fan</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x002B</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+                <requireCommand>IdentifyQuery</requireCommand>
```

Comment:
Is this correct? `Query` feature has conformance `!Matter` for Fan

Reply:
Right, updated